backends/base-backend.cpp

```diff
@@ -39,6 +39,20 @@ void BaseBackend::displayMessageOnOSD(const char *msg) {
 	dialog.runModal();
 }
 
+void BaseBackend::copyRectToOSD(const void *buf, int pitch, int x, int y, int w, int h) {
+	warning("BaseBackend::copyRectToOSD not implemented"); //TODO
+}
+
+void BaseBackend::clearOSD() {
+	warning("BaseBackend::clearOSD not implemented"); //TODO
+	//what should I do? Remove all TimedMessageDialogs?
```

Comment:
Yes, skip those. Additionally, we could add another backend feature: kFeatureOSD.